tensorflow/compiler/tf2tensorrt/convert/utils.h

```diff
@@ -73,6 +79,29 @@ struct VectorTensorShapeHasher {
    (NV_TENSORRT_MAJOR == major && NV_TENSORRT_MINOR == minor && \
     NV_TENSORRT_PATCH == patch && NV_TENSORRT_BUILD >= build))
 
+// This utility template converts an arithmetic type to a string.
+// This function is necessary to have the function:
+// `string DebugString(const std::vector<CType>&)` able to behave recursively.
+template<
+    typename CType,
+    typename = typename std::enable_if<
+      std::is_arithmetic<CType>::value, CType
+    >::type
+>
+string DebugString(const CType& el) {
+  string el_str = std::to_string(el);
+  el_str.erase(el_str.find_last_not_of('0') + 1, std::string::npos);
+  return el_str;
+}
+// This utility template converts nested vectors to a string for debug purposes.
+template <typename CType>
+string DebugString(const std::vector<CType>& vector){
+  string tmp_s = "";
+  for (const auto el: vector) {
```

Comment:
Will need a space between el and : to pass clang format check here.

Reply:
Fixed